src/background/adblocker.js

```diff
@@ -27,9 +27,73 @@ import asyncSetup from '/utils/setup.js';
 
 import { tabStats, updateTabStats } from './stats.js';
 import { getException } from './exceptions.js';
+import Config, {
+  FLAG_FIREFOX_CONTENT_SCRIPT_SCRIPTLETS,
+} from '/store/config.js';
 
 let options = Options;
 
+const contentScripts = (() => {
```

Comment:
same here, keeping the code in original position would make it easier to review

Reply:
I make position changes only when I think it is important - if we define variables, if it is possible, they should be defined before they are used by another part of the code.

This is PR, which touches this feature, so this is the best place to make such a change. For sake of visibility I'll make those changes in separate commits next time, so it will be easier to check logic change.

Reply:
two commits would work perfectly well, lets do that in the future